sox/file_info.py

```diff
@@ -21,17 +21,47 @@ def bitrate(input_filepath):
 
     Returns
     -------
-    bitrate : int
+    bitdepth : int
         number of bits per sample
         returns 0 if not applicable
     '''
     validate_input_file(input_filepath)
     output = soxi(input_filepath, 'b')
     if output == '0':
-        logger.warning("Bitrate unavailable for %s", input_filepath)
+        logger.warning("Bit depth unavailable for %s", input_filepath)
```

Comment:
Should this return `None` as well for consistency with bitdepth?

Reply:
Related - #78

Reply:
fixed in 7ddab5d